src/redux/rootReducer.ts

```diff
@@ -0,0 +1,21 @@
+import { Action, combineReducers } from '@reduxjs/toolkit';
+import { appVersionReducer } from './features/appVersion/appVersionSlice';
+
+// assembly of app reducers
+export const reducers = {
+  appVersion: appVersionReducer,
+};
+const combinedReducers = combineReducers(reducers);
+
+// used for building the typed root state
+type CombinedState = ReturnType<typeof combinedReducers>;
+
+export const rootReducer = (state: CombinedState | undefined, action: Action) => {
+  if (action.type === 'RESET_APP') {
```

Comment:
when/how do we use this?

Reply:
This is kinda a boilerplate you will see in a lot of stores. Its kinda the prototype first action that you have that you can use to reset the whole store. Its not normally used, but is just like a reset button on your PC